delivery/common/jwt/examples/jwt-hs256/main.js

```diff
@@ -0,0 +1,45 @@
+import { logger } from 'log';
+import { JWTValidator} from './jwt.js';
+import { crypto } from 'crypto';
+import { base16 } from 'encoding';
+
+//advanced options for jwt validator
+const jwtOption = {
+  //check token expiry
+  ignoreExpiration: false,
+  //check token nbf
+  ignoreNotBefore: false
+};
+
+const jwtValidator = new JWTValidator(jwtOption);
+
+export async function onClientRequest (request) {
+  try {
+    // Fetch hmac veification key from Propery Manager
+    const secretKey = request.getVariable('PMUSER_JWT_HMAC_KEY');
+    const sKey = await crypto.subtle.importKey(
+      'raw',
+      base16.decode(secretKey, 'Uint8Array').buffer,
+      {
+        name: 'HMAC',
+        hash: 'SHA-256'
+      },
+      false,
+      ['verify']
+    );
+    let jwt = request.getHeader('Authorization');
+    if (jwt){
+      jwt = jwt[0];
```

Comment:
I realize this is an example, but the [authorization header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization) has a leading `<auth-scheme>` string that it may be worth ignoring.

Reply:
Added logic to trim Bearer <auth-scheme> before passing to JWT validate function to indicate the token should be passed without any <auth-scheme> .